components/organisms/TitleSection.js

```diff
@@ -0,0 +1,24 @@
+import PropTypes from "prop-types";
+/**
+ *  page title section
+ */
+export default function TitleSection(props) {
```

Comment:
this is probably an atom, but I dont agree the the terminology anyways

Reply:
i think the horizontal rule would be an atom, this could be a molecule... as long as they are basically tiny, medium and large portions of code, it's ok.  I'll maybe move it though